README.md

```diff
@@ -97,65 +119,56 @@ on a connection provided by the pool will have the same effect as calling:
 
 ## Related Links
 
+- [NuoDB Multiplexer][5]
 - [NuoDB Node.js Driver Documentation][3]
 - [NuoDB Node.js Driver Docker Sample Express Application][4]
 
 ## Examples (EXPERIMENTAL)
 
 See branch [examples](https://github.com/nuodb/node-nuodb/tree/examples).
 
-## Help
-
-Issues and questions about node-nuodb can be posted on [GitHub][2].
+## Best Practices
 
-## Build and Test
+Any `try` `catch` block in which NuoDB resources are created must be followed by a `finally` block in which NuoDB resources are then cleaned up. Attempting to clean up NuoDB resources in a `try` block can lead to NuoDB resources being created an never cleaned up.
```

Comment:
Change "NuoDB resources being created an never cleaned up."
-> "NuoDB resources being created `and` never cleaned up."

Reply:
👍